entries/g/grab.com.json

```diff
@@ -0,0 +1,22 @@
+{
+  "Grab": {
+    "passwordless": "allowed",
+    "documentation": "https://engineering.grab.com/embracing-passwordless-authentication-with-passkey",
+    "regions": [
+      "sg",
+      "my",
+      "kh",
+      "id",
+      "mm",
+      "ph",
+      "vn",
+      "th"
+    ],
+    "categories": [
+      "food",
+      "payments",
+      "retail",
```

Comment:
I assume this is referring to GrabForBusiness' "sell with us"?

Reply:
GrabMart lets you purchase a variety of items from groceries to stationary (screenshot partially in Thai but hopefully you get the gist!)
![Screenshot_20250416-152053](https://github.com/user-attachments/assets/8489bc5e-8c92-42a0-8076-b260efe80eaf)